### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by jvaverka**

### CODE REVIEW

File: docs/src/fileformat.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ The following convenience "macros"/source placeholders are always expanded:
 
 - `@__NAME__`:
 
-  expands to the `name` keyword argument to [`Literate.markdown`](@ref),
```
Comment (fredrikekre) [inline docs/src/fileformat.md:136]: I don't agree with this one, the full sentence is
```
@__NAME__ expands to the name keyword argument
```
isn't that correct?

**Review by fredrikekre — COMMENTED** [review]
Thank you so much, it is nice to know someone reads the docs :)

File: docs/src/fileformat.md (hunk 1 above)
Comment (jvaverka) [reply docs/src/fileformat.md:136]: ah, yes that is correct - we should ignore my "fix" here

File: docs/src/fileformat.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ The following convenience "macros"/source placeholders are always expanded:
...
+  expands the `name` keyword argument to [`Literate.markdown`](@ref),
```
Comment (fredrikekre) [inline docs/src/fileformat.md:136]:
```suggestion
  expands to the `name` keyword argument to [`Literate.markdown`](@ref),
```

File: docs/src/fileformat.md (hunk 1 above)
Comment (jvaverka) [reply docs/src/fileformat.md]: Happy to help, the docs are awesome!

I'm planning to use Literate.jl for my Franklin.jl site, and the docs made it so easy.

Comment (fredrikekre) [reply docs/src/fileformat.md]: Good. I spent quite some time writing it, but you never really get much feedback so difficult to know if they are good or bad. Thanks again.

**Pull request merged**